website/src/contribute/release-guide.md

```diff
@@ -688,8 +688,20 @@ Verify that files are [present](https://dist.apache.org/repos/dist/dev/beam).
 * Build Python images and push to DockerHub.
 
 ```
-./gradlew :sdks:python:container:buildAll -Pdocker-tag=${RELEASE}_rc{RC_NUM}
+./gradlew :sdks:python:container:buildAll -Pdocker-pull-licenses -Pdocker-tag=${RELEASE}_rc{RC_NUM}
+```
+
+Verify that third party licenses are included by logging in to the images. For Python SDK images, there should be around 80 ~ 100 dependencies. 
+Please note that dependencies for the SDKs with different Python versions vary. 
+Need to verify all Python images by replacing `${ver}` in the following command to `python2.7, python3.5, python3.6, python3.7`.
```

Comment:
@ibzib , this part should be verified BEFORE pushing Python images. I'm not sure when this PR can be merged because of website issues, so explicitly pinging you here.

Reply:
Ack, thanks for the heads up